### PR TITLE
Ferris sweep

### PR DESCRIPTION
**Pull request by sorla**
Hi, this PR adds support for the Ferris Sweep keyboard. I hope the files are all in order.

PS. This is my first time using KMK and it's been a pleasure to get up and running with it. Thanks for all the work that's gone into it!

### CODE REVIEW

File: boards/ferris_sweep/kb.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     pins[4],  pins[5],  pins[6],  pins[7],  pins[8],
+                                  pins[10], pins[11],
+]
+
+_KEY_CFG_RIGHT = [
+    pins[19], pins[18], pins[17], pins[16],  pins[9],
+     pins[0], pins[12], pins[13], pins[14], pins[15],
+     pins[8],  pins[7],  pins[6],  pins[5],  pins[4],
+    pins[11], pins[10],                        
+]
```
Comment (xs5871) [inline boards/ferris_sweep/kb.py:27]: With the `coord_mapping` there's no reason to have left and right side pin definitions. They are identical, only mirrored.
Try reversing the rows of your right side, for example:
```python
coord_mapping = [
0, 1, 2, 3, 4,   21, 20, 19, 18, 17,
...]
```

File: boards/ferris_sweep/main.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+# fmt: off
+
+keyboard.keymap = [
+    [  # QWERTY
+    KC.Q,KC.W,  KC.E, KC.R,KC.T,  KC.Y, KC.U,KC.I,KC.O,KC.P,
+   A_SFT,S_L5,  D_L1, F_L3,KC.G,  KC.H, J_L4,K_L2,L_L6,SCLN_SFT,
+     KC.Z, X_CTL, C_ALT, KC.V,    KC.B,   KC.N, KC.M, COM_ALT, DOT_CTL,  KC.SLSH,
+        KC.LGUI,        KC.BSPC,SPC_L7,        KC.ENT,
+    ],
```
Comment (xs5871) [inline boards/ferris_sweep/main.py:71]: Some of your whitespace seems to got mangled here.

**Review by xs5871 — CHANGES_REQUESTED** [review]
Looks generally pretty decent, thanks for the contribution.
Got a couple of small improvements.

**Review by sorla — COMMENTED** [review]
Thanks for the feedback, I've made the changes suggested.

File: boards/ferris_sweep/kb.py (hunk 1 above)
Comment (sorla) [reply boards/ferris_sweep/kb.py]: That's a much better way of doing it! Thanks

File: boards/ferris_sweep/main.py (hunk 2 above)
Comment (sorla) [reply boards/ferris_sweep/main.py]: It did, thanks, should be fixed now

**Comment by xs5871** [comment]
Almost there. The linter is complaining:
It looks like the keymap isn't indented enough, or you may want to add a `flake8: noqa` to the `fmt: off`.

**Review by xs5871 — APPROVED** [review]

**Pull request merged**